qubesmanager/settings.py

```diff
@@ -648,6 +648,7 @@ def __init_advanced_tab__(self):
                 None,
                 allow_default=True, allow_none=True)
             self.kernel.currentIndexChanged.connect(self.kernel_changed)
+            self.kernel_opts.setText(getattr(self.vm, 'kernelopts', '[]'))
```

Comment:
Why `[]` as default value?

Reply:
for consistency - the [] was default content when there were no kernel opts before.